crates/paths/src/lib.rs

```diff
@@ -213,6 +213,13 @@ impl AbsPath {
     pub fn exists(&self) -> bool {
         self.0.exists()
     }
+
+    pub fn name_and_extension(&self) -> Option<(&str, Option<&str>)> {
+        Some((
+            self.file_stem()?.to_str()?,
+            self.extension().and_then(|extension| extension.to_str()),
+        ))
+    }
```

Comment:
This isn't a delegated method so we shouldn't put that into the `region` comment area, though assuming this is only in one place we might as well replace the function call with what we do here